ext/dcos-installer/dcos_installer/__init__.py

```diff
@@ -1,235 +1 @@
-            help='Hash a password on the CLI for use in the config.yaml.'
-        )
-
-        parser.add_argument(
-            '-v',
-            '--verbose',
-            action='store_true',
-            default=False,
-            help='Verbose log output (DEBUG).')
-
-        parser.add_argument(
-            '-p',
-            '--port',
-            type=int,
-            default=9000,
-            help=argparse.SUPPRESS)
-
-        parser.add_argument(
-            '--offline',
-            action='store_true',
-            default=False,
-            help='Do not install preflight prerequisites on CentOS7, RHEL7 in web mode'
-        )
-
-        mutual_exc.add_argument(
-            '--web',
-            action='store_true',
-            default=False,
-            help='Run the web interface.')
-
-        mutual_exc.add_argument(
-            '--genconf',
-            action='store_true',
-            default=False,
-            help='Execute the configuration generation (genconf).')
-
-        mutual_exc.add_argument(
-            '--preflight',
-            action='store_true',
-            default=False,
-            help='Execute the preflight checks on a series of nodes.')
-
-        mutual_exc.add_argument(
-            '--install-prereqs',
-            action='store_true',
-            default=False,
-            help='Install preflight prerequisites. Works only on CentOS7 and RHEL7.')
-
-        mutual_exc.add_argument(
-            '--deploy',
-            action='store_true',
-            default=False,
-            help='Execute a deploy.')
-
-        mutual_exc.add_argument(
-            '--postflight',
-            action='store_true',
-            default=False,
-            help='Execute postflight checks on a series of nodes.')
-
-        mutual_exc.add_argument(
-            '--uninstall',
-            action='store_true',
-            default=False,
-            help='Execute uninstall on target hosts.')
-
-        mutual_exc.add_argument(
-            '--validate-config',
-            action='store_true',
-            default=False,
-            help='Validate the configuration for executing --genconf and deploy arguments in config.yaml')
-
-        mutual_exc.add_argument(
-            '--test',
-            action='store_true',
-            default=False,
-            help='Performs tests on the dcos_installer application')
-
-        options = parser.parse_args(args)
-        level = 'INFO'
-        if options.verbose:
-            level = 'DEBUG'
-
-        coloredlogs.install(
-            level=level,
-            level_styles={
-                'warn': {
-                    'color': 'yellow'
-                },
-                'error': {
-                    'color': 'red',
-                    'bold': True,
-                },
-            },
-            fmt='%(message)s',
-            isatty=True
-        )
-
-        log.debug("Logger set to DEBUG")
-        return options
+__version__ = "1.7"
```

Comment:
Please just don't put it here at all. It isn't adding significant value, and the value in it is already incorrect / doesn't match the string we use in the UI